modules/integration_azure_firewall/conf/03-health-status.yaml

```diff
@@ -0,0 +1,27 @@
+module: firewall
+name: "Health state"
+
+transformation: false
+aggregation: false
+filtering: "filter('resource_type', 'Microsoft.Network/azureFirewalls') and filter('primary_aggregation_type', 'true')"
+tip: "https://docs.microsoft.com/en-us/azure/firewall/logs-and-metrics#metrics"
+
+
+signals:
+  signal:
+    metric: "FirewallHealth"
+
+# https://docs.microsoft.com/en-us/azure/firewall/logs-and-metrics#metrics
+rules:
+  critical:
+    threshold: 50
+    comparator: "=="
+    lasting_duration: "180s"
+    lasting_at_least: "== 1.0"
+
+  major:
+    threshold: 0
+    comparator: "=="
+    lasting_duration: "180s"
+    lasting_at_least: "== 1.0"
```

Comment:
define it to `== 1.0` results in string type which is ignored by template.

Moreover, define it as `1.0` is the same that do not define it at all.

I suggesst to remove this (same above)